.github/workflows/checks.yml

```diff
@@ -0,0 +1,88 @@
+name: Basic checks
+
+on:
+  pull_request:
+  push:
+    branches:
+      - master
+    tags-ignore: [ v.* ]
+
+jobs:
+  check-code-style:
+    name: Check Code Style
+    runs-on: ubuntu-18.04
+    steps:
+      - name: Checkout
+        uses: actions/checkout@v2
+        with:
+          # we don't know what commit the last tag was it's safer to get entire repo so previousStableVersion resolves
+          fetch-depth: 0
+
+      - name: Checkout GitHub merge
+        if: github.event.pull_request
+        run: |-
+          git fetch origin pull/${{ github.event.pull_request.number }}/merge:scratch
+          git checkout scratch
+
+      - name: Set up JDK 11
+        uses: olafurpg/setup-scala@v10
+        with:
+          java-version: adopt@1.11.0-9
+
+      - name: Cache Coursier cache
+        uses: coursier/cache-action@v5
+
+      - name: Code style check and binary-compatibility check
+        run: sbt "verifyCodeStyle; mimaReportBinaryIssues"
+
+  check-code-compililation:
```

Comment:
```suggestion
  check-code-compilation:
```
😉

Reply:
Thanks @ihostage. That typo come from the jdbc plugin. I copied from there. 
And it's probably in Alpakka Kafka as well. Because I copy from there a when I did the jdbc. But I may have included it myself at some point. I love some creative work! ;-)

Reply:
See, I told you.
https://github.com/akka/akka-persistence-jdbc/pull/519

Reply:
I really told you
https://github.com/akka/alpakka-kafka/pull/1346

🤦

Reply:
If you weren't a `member`, it would be a good chance to get a `contributor` badge for these repos 😂